src/Language/PureScript/Errors.hs

```diff
@@ -1195,6 +1197,15 @@ renderBox = unlines
   dropWhileEnd p = reverse . dropWhile p . reverse
   whiteSpace = all isSpace
 
+evalTypelevelFunctions :: Type -> Type
```

Comment:
There's something not quite right about this. Evaluating something of kind `Symbol` should result in a pretty printed string, but everything else should just show the regular `NoInstanceFound` error. Right now, if you have a concatenated string inside another type constructor, you'll get a mixture of the two.

I think you have to have a function of type `Type -> Maybe String` which tries to evaluate a string, and fails if it gets something else.